tests/console/check_system_info.pm

```diff
@@ -81,6 +81,20 @@ sub check_buildid {
     }
 }
 
+# SLE-23605: Python3 module
```

Comment:
I think you'd better add test steps in function description so that to make reviewers to understand your test better.

Reply:
+1